smpplib/consts.py

```diff
@@ -319,63 +298,60 @@
     )
 }
 
-
 STATE_SETTERS = {
     'bind_transmitter_resp': SMPP_CLIENT_STATE_BOUND_TX,
     'bind_receiver_resp': SMPP_CLIENT_STATE_BOUND_RX,
     'bind_transceiver_resp': SMPP_CLIENT_STATE_BOUND_TRX,
     'unbind_resp': SMPP_CLIENT_STATE_OPEN,
 }
 
-
 OPTIONAL_PARAMS = {
```

Comment:
I would suggest keeping old ids for readability
```python
OPTIONAL_PARAMS = {
    key: frozenset({value}) if not isinstance(value, Iterable) else frozenset(value)
    for key, value in six.iteritems({
#old values here,
'network_error_code': (0x0423, 0x1403,),
})}
```

Reply:
Author's option for me is more explicit, no need to read the conversion code. It's faster to visually skip commas than understand the loop

Reply:
Perhaps it is fine. I wouldn't insist

Actually what I was thinking about is to create a reverse of this mapping:
```python
    for key, value in six.iteritems(consts.OPTIONAL_PARAMS):
        if code in value
```
can be probably translated into something like `key = REV_OPTIONAL_PARAMS[code]`